contrib/seeds/README.md

```diff
@@ -5,8 +5,8 @@ Utility to generate the seeds.txt list that is compiled into the client
 
 The seeds compiled into the release are created from the current protx list, like this:
 
-```bash
-dash-cli protx list valid 1 2018966 > protx_list.json
+```sh
+dash-cli protx list valid 1 > protx_list.json
```

Comment:
you should be specifying specific block here?

Reply:
nope, by default it's "current tip"

Reply:
Yeah... Arguably we should be specifying a specific hash though so it's more reproducible

Reply:
You have to pick a specific block to make it reproducible. Could change that to accept block hashes but specifying block height is good enough if that block is chainlocked already.

Reply:
Idk, it's probably fine. We've had trouble actually reproducing these before

Reply:
why make it worse though? :)